Add spanish translation

danimiba opened the pull request:
Translated this article to spanish to reach a wider audience. As an Argentinian new media art student, I believe this is a key resource to have available for all my classmates. Hope you like it!

danimiba committed:
Add spanish translation

eviau-sat commented:
The last link gives me a 404

![image](https://user-images.githubusercontent.com/83365562/234331256-6c72a53c-cec2-438b-ada6-75df7f09eac1.png)

eviau-sat commented:
IT's this one : https://www-moma-org.translate.goog/explore/inside_out/2015/05/13/open-sourcing-momas-digital-vault/?_x_tr_sl=en&_x_tr_tl=es&_x_tr_hl=es

danimiba committed:
Fixed broken link

Linked it to the wayback machine as per suggested in https://github.com/antimodular/Best-practices-for-conservation-of-media-art/issues/6

danimiba committed:
Fixed broken link in sp version

As suggested here https://github.com/antimodular/Best-practices-for-conservation-of-media-art/issues/6

danimiba commented:
Thanks @eviau-sat for noticing it! Already fixed it <3

@stephanschulz I used the link you suggested in #6 

Please let me know if there's anything else!

danimiba committed:
Update readme.md

danimiba committed:
Fixed link and added image

danimiba committed:
Update fork and branch (#1)

* update markdown

* Update readme.md

update image link

* Update readme.md

update image url

* Update readme.md

* Update readme.md

* Update readme.md

* Create LICENSE.md

* Update LICENSE.md

* Update LICENSE.md

* Update readme.md

* Update readme.md

* Update readme.md

---------

Co-authored-by: Stephan Schulz <stephan@antimodular.com>

eviau-sat commented:
It looks good to me but do note that my Spanish is not advanced enough to really do copy writing / editing on this. I can try and find someone who might say yes to some light editing on this... let me know :)

eviau-sat commented:
I do have one tiny tiny suggestion for improving the internationlization of your efforts: renaming the file from `spanish-version.md` to `español-spanish.md` and/or `espagnol-spanish.md` ?

Why would you do this ?

If someone *only speaks Spanish*, they won't look for a file named spanish, they will look for a file named spanish-in-spanish (so espagnol / español ). this is the why of i10n (internationalization ) on wikipedia pages ! :) 

hope this helps, i trust your judgement on this one.

eviau-sat commented:
My last point - but this is a personal pet peeve of mine, no need to actually make these changes - the standard file for translation is a `.po` file.

So maybe moving from `.md` to `.rst` and then making a po file with Sphinx would make everything more manageable for future translators.

I can work on this part.

eviau-sat commented:
Ok - sorry for the stream of comments , hope it's the last for today ! : i can create the .po file and I can help you or do myself the moving from the `spanish-version.md` file to the `spanish-version.po` file - this might makes no sense if I say it that way.

What I mean is appreciate your work on translating this important work of conservation of media arts to spanish, and I am available to help you with the process as I can - so mostly on the technical side, not on the actual translation side of things.

Hope this is more clear ! Have a wonderful day :sparkles:  it's sunny/cloudy here for now ;)

stephanschulz commented:
I am sorry, right now I am super busy.
But we are very interested in having this in French, Spanish and languages. Also improving the presentation  and file formats is a good idea.
Maybe the two of you can collaborate on one repo that has all those improvements. Once done I will present it to Rafael for approval before merging it.
Would that work for you?

eviau-sat commented:
```
But we are very interested in having this in French, Spanish and languages. Also improving the presentation and file formats is a good idea.
Maybe the two of you can collaborate on one repo that has all those improvements. Once done I will present it to Rafael for approval before merging it.
Would that work for you?
```

Yes that would work for me if it works for @danimiba too ! I can start a repo with the `.rst` and `.po` files :) I can do that just now !

eviau-sat commented:
I have the .po files for fr and es ready to be translated/translitterated :sparkles: 

cannot push the repo because i get https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403 when i follow https://docs.github.com/fr/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token

FYI it looks like this wtih Sphinx

![image](https://user-images.githubusercontent.com/83365562/236212038-9ba38666-7e18-474d-844d-f8b34376e796.png)

danimiba commented:
Excellent! I usually don't work with translation files other than json or md, so I would greatly appreciate your help @eviau-sat !
If you'd like we can meet to discuss this on a call or something like that. Can we exchange emails?

And thank you @stephanschulz for the offer! We will try to make a cohesive and robust proposal for future translations ❤️

eviau-sat commented:
We can exchange emails if you can guess mine ;) I work at sat.qc.ca !

eviau-sat commented:
I am going to live-try https://github.com/mondeja/mdpo - wish me luck :)

eviau-sat commented:
It gives me something like this: https://gist.github.com/eviau-sat/1145fe734a7d4516b1c5be786d88c6b7

github auth is making me go in circles again so i will leave it at that for now... :)

eviau commented:
hello @danimiba :)

sorry for the lack of progress on this - as i mentionned in French in #7 i got side-tracked by figuring out the best way to go about this. this is my personal github account.

**My current suggestion - also described in #7 - is to have `.md` file, one by locale, and to build a Jekyll website from there.**

The website should look like this: http://eviau.net/Best-practices-for-conservation-of-media-art/

The repo will look like this: https://github.com/eviau/Best-practices-for-conservation-of-media-art/tree/just-jekyll

I've tried various set-ups but they all seem too cumbersome for the current task at hands. I am happy to describe them if needed!

My thoughts are that the original text will probably not change enough to justify setting up a process to update `.po` files on the regular. So `.md` files are more than enough for this for now.

If this makes sense to everyone, I think this means that:

- the name of this Spanish translation file should be changed to `readme-es.md` to standardize with usual locales
- someone on the antimodular team - @stephanschulz perhaps - could activate a jekyll website by going in Settings>Pages and selecting _Deploy from a branch_ and selecting the `master` branch of this repo. This will make the original readme available - I doubt it will be at antimodular.github.io/Best-practices-for-conservation-of-media-art but it might be if there is a repo named antimodular.github.io with Pages activated ?

and then we can have PRs for every locale someone would like to translate to :)

Let me know what you think!

stephanschulz commented:
I did this "Settings>Pages and selecting Deploy from a branch and selecting the master branch of this repo"

![Screen Shot 2023-08-19 at 11 10 34 AM](https://github.com/antimodular/Best-practices-for-conservation-of-media-art/assets/1054816/a9c2a7e3-dad2-4d90-a4a0-7ce5df3e4fa5)

eviau commented:
Seems like it worked : http://antimodular.github.io/Best-practices-for-conservation-of-media-art/

stephanschulz commented:
@danimiba @eviau did you manage to translate this document in to spanish and french? i would like to show those to Rafael.
thank you,
stephan

danimiba commented:
Hi @eviau and @stephanschulz !

Sorry I haven't been able to help with the technical aspect of the management of the different languages in the page. It is a bit out of my scope and I haven't had the time to learn.

As of the spanish translation, it is already done in this same PR, you can check the File changes. If you need me to rename it, or change it in any way to help with the page, please let me know. Also, if you need the PDF I can send it by mail.

Thank you for everything,
Dani

eviau commented:
Hello @stephanschulz !

I am about 1/3 into the fr translation. I can make a PR over the week-end.
